tests_integ/test_max_tokens_reached.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from strands import Agent, tool
+from strands.models.bedrock import BedrockModel
+from strands.types.exceptions import MaxTokensReachedException
+
+
+@tool
+def story_tool(story: str) -> str:
+    return story
+
+
+def test_context_window_overflow():
+    model = BedrockModel(max_tokens=1)
```

Comment:
I think this needs to be higher. If we look at the generated output, with only one token, the model wont event be able to send a single toolUse event (which is the source of this error)
```suggestion
    model = BedrockModel(max_tokens=100)
```